test/Microsoft.AspNetCore.SignalR.Redis.Tests/RedisHubLifetimeManagerTests.cs

```diff
@@ -552,7 +553,7 @@ public class TestObject
 
         private RedisHubLifetimeManager<MyHub> CreateLifetimeManager(TestRedisServer server, MessagePackHubProtocolOptions messagePackOptions = null, JsonHubProtocolOptions jsonOptions = null)
         {
-            var options = new RedisOptions() { Factory = t => new TestConnectionMultiplexer(server) };
+            var options = new RedisOptions() { Factory = async (t) => await Task.FromResult(new TestConnectionMultiplexer(server)) };
```

Comment:
Can be: `Factory = (t) => Task.FromResult<IConnectionMultiplexer>(new TestConnectionMultiplexer(server))`